package.json

```diff
@@ -56,11 +56,12 @@
     "@babel/plugin-transform-object-assign": "^7.14.5",
     "@babel/plugin-transform-runtime": "^7.15.0",
     "@babel/register": "^7.15.3",
+    "@babel/runtime": "^7.15.4",
     "@emotion/cache": "^11.4.0",
     "@emotion/react": "^11.4.1",
     "@emotion/styled": "^11.3.0",
     "@eps1lon/enzyme-adapter-react-17": "^0.1.0",
-    "@material-ui/monorepo": "https://github.com/mui-org/material-ui.git#next",
+    "@material-ui/monorepo": "https://github.com/m4theushw/material-ui.git#master",
```

Comment:
Once we're okay with the shape of this PR, I'll open another one in the core with the changes from my fork. Basically what I did was to move code from the core's script to separate files to reuse here. Here's the diff: https://github.com/mui-org/material-ui/compare/master...m4theushw:master